customext/dummy_rocc.cc

```diff
@@ -44,4 +44,4 @@ class dummy_rocc_t : public rocc_t
   reg_t acc[num_acc];
 };
 
-REGISTER_EXTENSION(dummy_rocc, []() { return new dummy_rocc_t; })
+REGISTER_EXTENSION(rocc, []() { return new dummy_rocc_t; })
```

Comment:
This should not be part of the same commit, nor even the same PR.

Reply:
Got it, I will remove this.